### PR TITLE
just some small refactoring

### PR DESCRIPTION
**Pull request by guusvw**

### CODE REVIEW

File: pkg/controller/labels.go [hunk 1]
```diff
@@ -41,7 +42,7 @@ func getLabelsToEnsureFromMessage(message string) []string {
 
 func labelSliceContains(slice []*github.Label, s string) bool {
 	for _, element := range slice {
-		if element.GetName() == s {
+		if strings.Compare(element.GetName(), s) == 0 {
```
Comment (alvaroaleman) [inline pkg/controller/labels.go]: Why did you change this? From the `go doc`:

```
Compare is included only for symmetry with package bytes. It is usually
    clearer and always faster to use the built-in string comparison operators
    ==, <, >, and so on
```

**Review by alvaroaleman — CHANGES_REQUESTED** [review]

File: pkg/controller/labels.go (hunk 1 above)
Comment (guusvw) [reply pkg/controller/labels.go]: what? ;)  - fixed

**Review by alvaroaleman — APPROVED** [review]
